runtime/common/src/lib.rs

```diff
@@ -37,6 +43,22 @@ macro_rules! production_or_benchmark {
 	}};
 }
 
+pub struct CurrencyEDs<T>(PhantomData<T>);
```

Comment:
A currency has one associated existential deposit, not multiple. In `orml_tokens` it is plural because it's called `ExistentialDeposits` (i.e, the struct that handles all the existential deposits). This struct we are adding here is "Currency Existential Deposits" which, again, is semantically incorrect. In any case, even if `orml_tokens` named it `CurrencyEDs`, we shouldn't propagate bad naming.

This is a nit and not a blocker at all.

Reply:
I think if you see it as:
- Thing that given a currency it returns a ED, then `CurrencyED`
- Thing that stores all EDs for each currency. You see it as a whole, then `CurrencyEDs`

It reminds me to Haskell partial applications 😄:
- `Storage -> CurrencyID -> Balance` // Receives 2 parameters and returns a value
- `Storage -> (CurrencyID -> Balance)` // Receives 1 parameter and returns a function that receives 1 parameter and returns a value.

And actually both things are the same, because the parenthesis is optional. So I would say that it depends of the PoV of such parenthesis 🤣. But yeah, I can update it. I have no strong opinion from my side.

Reply:
Done!